qpc_tools/install/cli.py

```diff
@@ -31,6 +31,27 @@ def __init__(self, subparsers):
         # pylint: disable=no-member
         CliCommand.__init__(self, self.SUBCOMMAND, self.ACTION,
                             subparsers.add_parser(self.ACTION))
+        self.parser.add_argument('--offline', dest='offline',
+                                 choices=install.BOOLEAN_CHOICES,
+                                 default='false',
+                                 help=_(messages.CLI_INSTALL_OFFLINE_HELP),
+                                 required=False)
+        self.parser.add_argument('--version', dest='version',
+                                 default='latest',
+                                 help=_(messages.CLI_INSTALL_VERSION_HELP),
+                                 required=False)
+        self.parser.add_argument('--home-dir', dest='home_dir',
+                                 default='latest',
```

Comment:
So I am sorry that I didn't catch this the first round of reviews, but do we want the default for the `--home-dir` to be `latest` instead of `~/quipucords`.

Reply:
Fixed.  Nice catch Cody.